doc/7/controllers/security/get-user-strategies/snippets/get-user-strategies.js

```diff
@@ -0,0 +1,10 @@
+try {
```

Comment:
IMHO you could remove the `try...catch` because it does not bring any more value to the snippet

Reply:
I was just following the global snippet architecture.